apps/frontend/src/libs/enums/icon-name.enum.ts

```diff
@@ -0,0 +1,6 @@
+const IconName = {
```

Comment:
We already have type IconName, we don't need enum

Reply:
so I can just pass the name as a string, not a constant? 
```
iconName={isPasswordVisible ? "crossedEye" : "eye"}
```
Or is it better to leave the enum and change the type to this?
```
import { ICON_NAMES } from "~/libs/enums/enums.js";

type IconName = typeof ICON_NAMES[keyof typeof ICON_NAMES];
```